runtime/tutor

```diff
@@ -488,7 +491,8 @@ _________________________________________________________________
 
  Type C to duplicate the cursor to the next suitable line.
 
- 1. Move the cursor to the first line marked '-->' below.
+ 1. Move the cursor to the first line below marked '-->'. Place
+    the cursor on the first word.
```

Comment:
This change shouldn't be necessary; there is no need for the cursor to be placed specifically on the first word in this example.

Reply:
It must be past the '-->'. This is the simplest way to make that happen.

Reply:
But I see your point that it could confuse the reader, giving the impression that it *has* to start on the first word. A better replacement might be "Place the cursor somewhere after the '-->'."

Reply:
I changed it to "Place the cursor somewhere past the '-->'." I hope that conveys the right message.